lib/puppet/face/node/clean.rb

```diff
@@ -57,11 +57,11 @@ def inform(message)
 
   # clean signed cert for +host+
   def clean_cert(node)
-    begin
-      require 'puppetserver/ca/cli'
+    if Puppet.features.puppetserver_ca?
+      require 'puppetserver/ca/action/clean'
       Puppetserver::Ca::Action::Clean.new(LoggerIO.new).run({ 'certnames' => [node] })
-    rescue LoadError => e
-      Puppet.warning _("Unable to clean up certs for %{node}: %{error}") % { node: node, error: e }
+    else
+      Puppet.info _("Not managing %{node} certs as this host is not a CA") % { node: node }
```

Comment:
Went back to the original error here, so that it can stay translated (we're post-string freeze). Still seems relevant if someone is trying to run this on an agent node that doesn't have the gem.